x-pack/plugins/stack_connectors/common/crowdstrike/schema.ts

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { SUB_ACTION } from './constants';
+
+// Connector schema
+export const CrowdstrikeConfigSchema = schema.object({});
+export const CrowdstrikeSecretsSchema = schema.object({
+  clientId: schema.string(),
+  clientSecret: schema.string(),
+});
+
+// TODO TC: Response Schemas - most is commented out, because we are not able to test it against a real endpoint just yet.
```

Comment:
suggestion:

If the data you have commented out below is "real" data (meaning: it was defined based on the data you captured while you had access to the customer's environment), then I would actually uncomment it all - we need to have something to work with as we move forward to mock a crowdstrike API server.

For reponse data properties that we don't forsee needing or for those that you are not sure are always returned, I would say wrap them in `schema.nullable()`. That along with the options to allow unknowns should hopefully prevent the Sub-actions framework from emitting response schema validation failures.

Reply:
should probably remove this `todo` now